tests/tasks/get_profile_stat.yml

```diff
@@ -45,12 +48,26 @@
         grep "^# Ansible managed"
         /etc/sysconfig/network-scripts/ifcfg-{{ profile }}
       register: _result
+      changed_when: false
 
     - name: Verify the ansible_managed comment in ifcfg-{{ profile }}
       set_fact:
         lsr_net_profile_ansible_managed: true
       when:
         - _result.stdout_lines | length == 1
+
+    - name: Get the fingerprint comment in ifcfg-{{ profile }}
+      command: >-
+        grep "^# system_role:network"
+        /etc/sysconfig/network-scripts/ifcfg-{{ profile }}
+      register: _result
+      changed_when: false
+
+    - name: Verify the fingerprint comment in ifcfg-{{ profile }}
+      set_fact:
+        lsr_net_profile_fingerprint: true
+      when:
+        - _result.stdout_lines | length == 1
   when:
```

Comment:
ansible-lint 6.x suggests that the conditional `when` should come before the `block`.